src/js/angular/core/directives.js

```diff
@@ -343,37 +346,36 @@ function searchResourceInput($location, toastr, ClassInstanceDetailsService, Aut
                 return $sce.trustAsHtml(resultItem.description);
             };
 
-            $scope.searchRdfResource = function (uri, callback) {
-                if (uri.type === 'prefix') {
-                    $scope.searchInput = expandPrefix(uri.value + ':');
+            $scope.searchRdfResource = function (resource, callback) {
+                if (resource.type === 'prefix') {
+                    $scope.searchInput = expandPrefix(resource.value + ':');
                     $scope.autoCompleteUriResults = [];
                 } else {
-                    let textUri;
-                    if (angular.isUndefined(uri)) {
-                        textUri = $scope.searchInput;
+                    let textResource;
+                    if (angular.isUndefined(resource)) {
+                        textResource = $scope.searchInput;
                     }
-                    if (typeof uri === 'object') {
-                        textUri = uri.value;
+                    if (typeof resource === 'object') {
```

Comment:
else if maybe?

Reply:
otherwise textResource will become undefined when resource is undefined

Reply:
Fixed